io_scene_nif/modules/animation/animation_import.py

```diff
@@ -149,7 +138,7 @@ def add_key(self, fcurves, t, key, interp):
         """
         Add a key (len=n) to a set of fcurves (len=n) at the given frame. Set the key's interpolation to interp.
         """
-        frame = round(t * self.fps)
+        frame = round(t * animation.FPS)
```

Comment:
Not sure if this should be an class variable. Seems hacky idk. I would have all frames > seconds conversions use a function in a common scene or animation instance, one we make the anim modules inherit from.

Reply:
Yeah I was calling it once as a static method of the global scene.
Main motivation was not having instance methods for now.